src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadataV2Feed.cs

```diff
@@ -13,33 +14,6 @@ namespace NuGet.Protocol
 {
     public class PackageSearchMetadataV2Feed : IPackageSearchMetadata
     {
-        public PackageSearchMetadataV2Feed(V2FeedPackageInfo package)
```

Comment:
Deleting a public constructor on a public class can break customers who use this package. Is there a reason why it's deleted?

Reply:
No, it is just not used anymore within `NuGet.Client` but I can add it back.